packages/labs/src/components/tag-input/tagInput.tsx

```diff
@@ -303,9 +333,23 @@ export class TagInput extends AbstractComponent<ITagInputProps, ITagInputState>
                 this.handleBackspaceToRemove(event);
             }
         }
+
+        // stop the event from propagating to the container's onKeyDown
+        // callback. we'll invoke the top-level callback directly with an
+        // `undefined` index.
+        event.stopPropagation();
```

Comment:
might be safer to `preventDefault()` and check `isDefaultPrevented()` in `handleContainerKeyDown`?